syntax = "proto3";

package ovms_ipc;

message NumpyAttributes{
    repeated int32 shape = 1;
    string data_type = 2;
}
message StringAttributes{
    int32 length = 1;
    string encoding = 2;
}
message BinaryAttributes{
    int32 length = 1;
}
